specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2019-03-01/edgegateway.json

```diff
@@ -520,45 +520,6 @@
       }
     },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataBoxEdge/dataBoxEdgeDevices/{deviceName}/getExtendedInformation": {
```

Comment:
@anponnet - Has this API ever worked? It couldn't have since the URL is not well-formed as per ARM guideline. Was the PUT added by accident to swagger? Or do you actually have a PUT for this path? 
I checked the calls to this API and there is no traffic to this path in the last 45 days. 
Please confirm and then we can sign off. Even though theoretically its breaking change for the SDK, but my hunch is that it never would have worked. And so it may be ok.

Reply:
@ravbhatnagar , We had a PUT call on this path when we released our private preview which was not reviewed by ARM. Later on this API was removed and it is never in use. So there wont be any call to that API. Though API was removed, the Path was part of swagger and it went in our last swagger release. Our SDK (or even API documentation) is not yet released, so it is safe to remove this

Reply:
Sounds good and thanks for confirming. @praries880 - this should be good to remove.